packages/flutter/test/material/scrollbar_test.dart

```diff
@@ -988,7 +990,9 @@ void main() {
   testWidgets('Hover animation is not triggered by tap gestures', (WidgetTester tester) async {
     await tester.pumpWidget(
       MaterialApp(
-        theme: ThemeData(scrollbarTheme: ScrollbarThemeData(
+        theme: ThemeData(
+          useMaterial3: false,
+          scrollbarTheme: ScrollbarThemeData(
```

Comment:
The indent below ScrollbarThemeData got dropped. Here and elsewhere in this file.